plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -31,6 +31,7 @@ import kotlinx.coroutines.Deferred
 import kotlinx.coroutines.Dispatchers
```

Comment:
Commit message nit: "This commit *enforces*...". Or maybe something like: "This commit adds concurrent processing again."